Dockerfile

```
@@ -1,22 +1,28 @@
-FROM java:7
+FROM ubuntu:20.04 AS builder
 
-ENV RELEASE_DATE 2017-04-23
-ENV H2DATA /h2-data
+ARG RELEASE_DATE=2022-06-13
+ARG RELEASE_VERSION=2.1.214
+
+RUN apt-get update \
+  && apt-get install curl -y \
+  && apt-get install unzip -y \
+  && curl -L https://github.com/h2database/h2database/releases/download/version-${RELEASE_VERSION}/h2-${RELEASE_DATE}.zip -o h2.zip \
+  && unzip h2.zip -d . \
+  && cp h2/bin/h2-${RELEASE_VERSION}.jar /h2.jar
 
-RUN curl http://www.h2database.com/h2-$RELEASE_DATE.zip -o h2.zip \
-    && unzip h2.zip -d . \
-    && rm h2.zip
+FROM eclipse-temurin:8-jre
```

It's probably better to explicitly set a version tag and use Ubuntu Focal
```suggestion
FROM eclipse-temurin:8u372-b07-jre-focal
```